README.md

```diff
@@ -1,17 +1,175 @@
+doesn't scale to a billion users!
+
+This means that, sometimes, incomplete solutions can remain in place for years,
+or even decades, while the maintainers go back and forth trying to find a better
+approach that won't open an eldritch portal to Bosch's [Garden of Earthly
+Delights][earthly_delights].
+
+[earthly_delights]: https://www.khanacademy.org/humanities/renaissance-reformation/northern/hieronymus-bosch/a/bosch-the-garden-of-earthly-delights
+
+
+# What's all this have to do with `DELETE` requests?
+
+As of HTML5, forms officially do not support `DELETE` and `PATCH` for their
+methods.
+
+There's no short and sweet answer to explain this. If you want to dive deep and
+understand as much as possible about the decisions that went into it, you can
+read more on [this illuminating StackExchange post][why_no_put_delete], but for
+the purposes of succinct explanation, you can always stick with the tried and
+true "for historical reasons".
+
+[why_no_put_delete]: http://programmers.stackexchange.com/questions/114156/why-are-there-are-no-put-and-delete-methods-on-html-forms
+
+What you're seeing in the above `#form_tag()` behavior is a **workaround**
+implemented for us by Rails itself. With this in mind, we get the best of both
+worlds:
+
+- We get to be **good HTTP-abiding citizens** who use the correct request
+  methods for their corresponding goals (`GET` for read, `PATCH` for update, and
+  so on).
+- We get to **maintain our sanity** and not worry about W3C drama while writing
+  views.
+
+
+# That's great. Can we actually delete something now?
+
+Thus enlightened, we can (finally) proceed with our original goal:
+
+```ruby
+# app/controllers/people_controller.rb
+
+  def destroy
+    Person.find(params[:id]).destroy
+    redirect_to people_url
+  end
+```
+
+Nothing too special happening here, except for a bit of method-chaining to
+immediately destroy the instance we find.
+
+
+# Fancy JavaScript Helper
+
+As shown, you have to go to a user's `show` page to delete them. What if we want
+an admin control panel where users can be deleted from a list?
+
+```erb
+<!-- app/views/people/index.html.erb //-->
+
+<% @people.each do |person| %>
+<div class="person">
+  <span><%= person.name %></span>
+  |
+  <%= link_to "Delete", person, method: :delete, data: { confirm: "Really?" } %>
+</div>
+<% end %>
+```
+
+[`link_to`][url_helper_link_to] is a method of `UrlHelper` that has a number of
+convenient features. It describes the `:method` option as follows:
+
+[url_helper_link_to]: http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to
+
+> This modifier will dynamically create an HTML form and immediately submit the
+> form for processing using the HTTP verb specified.
+
+This uses JavaScript, and it will use `GET` (the default method used by all
+browsers for HTML links) if the user has JavaScript disabled.
+
+This example also uses the `:data` option to make sure the user is ready to
```

Comment:
What happens when you include this data option? I'd mention how it pops up a confirmation window.